### DISCUSSION TITLE
Contact Support <a href> link needed in Payment Page

### DISCUSSION THREAD

**Discussion by mar101xy** [btcpayserver/btcpayserver#6495] [Ideas / Feature Requests]
I have noticed a lot of random customers from btcpay server instances visiting the btcpay Telegram group asking for support to problems like their payment didn't go through or something else.

Customers then get confused and will click on the "Powered by BTC Pay" logo thinking that's where they need to go to get help.

I think a href link or custom message to contact support of that btcpay instance below the invoice of the payment page would be helpful.

I think this is important as I've have also been told that many customers get the most errors like "no route" when trying to pay using lightning since it's still fairly new.

This would also help the btcpay team as they are dealing with random customers asking for support 

![custom-message](https://github.com/user-attachments/assets/984ac970-4701-42ed-87e9-5fba7d587f5f)

**Comment by dennisreimann** [comment]
On the Store Settings > Checkout page, one can already define a "Support URL":

![grafik](https://github.com/user-attachments/assets/e76f5ae9-c5a0-44b4-af31-bab24350fbf9)

This is shown on the Invoice Expired page, but not in general. Maybe we should generalize this. /cc @pavlenex @dstrukt

**Reply by dstrukt** [reply]
Yeah that's probably a reasonable approach, similar to branding, this is a Server wide configuration, that if it contains the URL probably makes sense to display on all of the various public facing pages, unless explicitly disabled on specific pages.

Think we could also remove that first bit `For support requests related to partially paid invoices.`, and combine it into .. `A "Contact Us" support link will be shown...` to reduce a bit of text and redundancy.

**Reply by dstrukt** [reply]
Thanks for the feedback @mar101xy!

**Comment by mar101xy** [comment]
this has been implemented in v2.05
I upgraded and tried it and it works greats
thanks BTCPAY team

https://github.com/btcpayserver/btcpayserver/releases/tag/v2.0.5

closing the discussion
